src/lib/misc/recommended-styles.css

```diff
@@ -37,3 +37,9 @@
   background-color: rgb(var(--m3-scheme-tertiary-container));
   color: rgb(var(--m3-scheme-on-tertiary-container));
 }
+
+:root {
+  scroll-behavior: smooth;
+  scrollbar-color: rgb(var(--m3-scheme-on-secondary-fixed-variant))
+    rgb(var(--m3-scheme-surface-container-low));
+}
```

Comment:
try to make each pr only focus on one thing (eg one pr per feature for small features, or one pr per step for large features that require refactoring or exposing utilities)

Reply:
"with the website design changes" the website design changes should be relevant to the component